spring/src/test/java/io/seata/spring/annotation/MethodDescTest.java

```diff
@@ -30,13 +33,28 @@
     private static final GlobalTransactionScanner GLOBAL_TRANSACTION_SCANNER = new GlobalTransactionScanner(
         "global-trans-scanner-test");
     private static Method method = null;
+    private static Class<?> targetClass = null;
     private static GlobalTransactional transactional = null;
 
     public MethodDescTest() throws NoSuchMethodException {
         method = MockBusiness.class.getDeclaredMethod("doBiz", String.class);
         transactional = method.getAnnotation(GlobalTransactional.class);
     }
 
+    @Test
+    public void testGetAnnotation() throws NoSuchMethodException {
```

Comment:
1. method, class both has Annotation
2.a. only method has Annotation, class is not null
2.b. method has Annotation, class is null
3.a. only class has Annotation, method is not null
3.b. only class has Annotation, method is null
lack 2.a and 3.a